lib/import/brca/scripts/bash/Import_all_brca_interactive.sh

```diff
@@ -178,19 +178,32 @@ $BRAKE import:brca fname="$(echo "$x" | sed -e 's:.*pseudonymised_data/\(.*\):\1
 done
 }
 
-
+#to handle the new file format
 RJ7 () {
 MBIS=$1
 PROV='RJ7'
 IFS=$'\n'
+for x in $(find $DIRPATH/$FILEPATH -type f -name "*.pseudo" -path "*/$PROV/*" -name "*HBOC*")
+do
+IFS="$OIFS"
+$BRAKE import:brca fname="$(echo "$x" | sed -e 's:.*pseudonymised_data/\(.*\):\1:')" prov_code=$PROV
+done
+}
+
+#to handle the old file format
+RJ7_2 () {
+MBIS=$1
+PROV='RJ7'
+IFS=$'\n'
 for x in $(find  $DIRPATH/$FILEPATH -type f -name "*.pseudo" -path "*/$PROV/*" \
 -not -path "*/2021/*" \
 -not -path "*/2022/*" \
 -not -path "*/2023/*" \
-! -name "c466c80823235315f4df98bb4a14c4937ee5cbc4_08.2020_STG HBOC PHE reported till 28082020.xlsx.pseudo")
+! -name "c466c80823235315f4df98bb4a14c4937ee5cbc4_08.2020_STG HBOC PHE reported till 28082020.xlsx.pseudo" \
+! -name "*HBOC*")
 do
 IFS="$OIFS"
-$BRAKE import:brca fname="$(echo "$x" | sed -e 's:.*pseudonymised_data/\(.*\):\1:')" prov_code=$PROV
+$BRAKE import:brca fname="$(echo "$x" | sed -e 's:.*pseudonymised_data/\(.*\):\1:')" prov_code='RJ7_2'
```

Comment:
Hard code prov code should be removed.

Reply:
see [8df17eb](https://github.com/NHSDigital/data_management_system/pull/77/commits/8df17ebc5b5bd20b2bc35fa463d0db292be166c6)